task1/js/app.js

```diff
@@ -0,0 +1,46 @@
+let totalZombies = 0;
+let deadZombies = 0;
+let smallZombies = 0;
+let madZombies = 0;
+let strongZombies = 0;
+
+const totalSelector = document.querySelector('.total-zombies');
+const deadSelector = document.querySelector('.dead-zombies');
+const smallSelector = document.querySelector('.small-zombies');
+const madSelector = document.querySelector('.mad-zombies');
+const strongSelector = document.querySelector('.strong-zombies');
+
+for (const i in zombiesData) {
+    const currentZombie = zombiesData[i];
+    const isNumber = !isNaN(currentZombie);
+    const isDead = !isNumber || currentZombie < 1;
+    const isSmall = isNumber && (currentZombie > 0 && currentZombie < 11);
+    const isMad = isNumber && (currentZombie > 10 && currentZombie < 21);
+    const isStrong = isNumber && currentZombie > 20;
+
+    if (isDead) {
+        deadZombies++;
+    } else if (isSmall) {
+        smallZombies++;
+    } else if (isMad) {
+        madZombies++;
+    } else if (isStrong) {
+        strongZombies++;
+    }
+
+    totalZombies++;
+}
+
+const total = `${totalSelector.textContent} {${totalZombies}}`;
```

Comment:
"total", "dead", ...
is not a valid name for this
it's better using naming like "totalText", "deadText", ...